### PR TITLE
Include messageUs data in dcr model

### PR DESCRIPTION
**Pull request by sophie-macmillan**
## What does this change?
This updates the DCR model for liveblogs, to include the message us data.

## Does this change need to be reproduced in dotcom-rendering ?
- [ ] No
- [x] Yes (please indicate your plans for DCR Implementation)
[DCR will need to be updated to consume the messageUs data, and render the messageUs component accordingly](https://github.com/orgs/guardian/projects/80/views/1?pane=issue&itemId=19904585)

## Screenshots
image

messageUs value is present on code, when the message us data is present in s3 for this article

image
After removing the object from s3, the messageUs value is no longer present

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [x] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [x] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [x] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [n/a] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [n/a] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [n/a] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

File: article/test/LiveBlogControllerTest.scala [hunk 1]
```diff
@@ -287,6 +298,52 @@ import services.newsletters.{NewsletterApi, NewsletterSignupAgent}
...
+      isLivePage = Some(true),
+      filterKeyEvents = Some(false),
+      topics = None,
+    )(fakeRequest)
+    status(result) should be(200)
+
+    val content = contentAsString(result)
+    content should not include ("\"messageUs\":{\"formId\":\"mock-form-id\"}")
+  }
+
```
Comment (marjisound) [inline article/test/LiveBlogControllerTest.scala:346]: Nice test coverage 👍

**Review by marjisound — APPROVED** [review]
Amazing work, looks great 👍

File: article/app/services/MessageUsService.scala [hunk 2]
```diff
@@ -36,8 +36,8 @@ class MessageUsService(messageUsS3Client: S3Client[MessageUsConfigData]) extends
       }
   }
 
-  def getBlogMessageUsConfigData(blogId: String): Option[MessageUsConfigData] = {
-    messageUsConfigData.get().flatMap(_.get(blogId))
+  def getBlogMessageUsConfigData(blogId: String): Option[MessageUsData] = {
```
Comment (georgeblahblah) [inline article/app/services/MessageUsService.scala:39]: I wondered if this method should be renamed to align better to the return type? Something like `getBlogMessageUsData` maybe?

**Review by sunnyclimber456 — APPROVED** [review]
Looks good! I had one non-blocking comment about a function rename to align to its return type more closely

**Pull request merged**